Issue #349: test readme.md to be updated (#355)
dotkernel/dot-admin: README.md

composer install
```

> You will get a prompt like this:

Reviewer: ```suggestion
You will be prompted with the below message to choose whether you want to inject ConfigProviders:
```

```shell
 Please select which config file you wish to inject 'Laminas\Validator\ConfigProvider' into:
  [0] Do not inject
  [1] config/config.php
  Make your selection (default is 1):
```

> Type 0 to select **[0] Do not inject**.

Reviewer: ```suggestion
Type `0` to select **[0] Do not inject** and hit `Enter`.
```

> No other Config Providers need to be injected.

Reviewer: ```suggestion
```

> We choose 0 because Dotkernel includes its own ConfigProvider which already contains the prompted configurations.

Reviewer: ```suggestion
We choose not to inject any ConfigProvider because Dotkernel Admin comes with all the required ConfigProviders already injected in `config/config.php`.
```

> If you choose **[1] config/config.php**, an extra ConfigProvider will be injected.

Reviewer: ```suggestion
Choosing to inject any extra ConfigProvider would cause having duplicates which are not allowed and would crash the application.
```

### Test the installation

> If you are using virtual hosts as described in the [Dotkernel documentation] (https://docs.dotkernel.org/development/) you need you modify the permissions of the **data** and **log** folders:

Reviewer: ```suggestion
If you are using virtual hosts as described in the [Dotkernel documentation] (https://docs.dotkernel.org/development/) you need you modify the permissions of the `data`, `public/uploads` and `log` folders:
```

```shell

> chmod 777 data/

Reviewer: ```suggestion
chmod -R 777 data
```

> chmod 777 log/

Reviewer: ```suggestion
chmod -R 777 public/uploads
chmod -R 777 log
```

> chmod 777 data/cache/

Reviewer: Have you tested that besides `data` `data/cache` also needs to be `chmod`ed?
